packages/pyo3_samples/src/src/lib.rs

```diff
@@ -0,0 +1,232 @@
+extern crate serde;
```

Comment:
Couldn't we load it as a zip from some pinned commit in https://github.com/saidvandeklundert/pyo3 directly?

Reply:
Yeah we can switch to that, though currently I've modified it to remove the broken functions so we'd maybe need a commit on my fork. For experimenting, it's more convenient for it to be in-tree though.